src/main/java/com/simplythewest/utd/priorityList/ui/MainMenu.java

```diff
@@ -1,15 +1,17 @@
-package com.simplythewest.utd;
+package com.simplythewest.utd.priorityList.ui;
 
+import com.simplythewest.utd.priorityList.models.ToDoItemRepository;
 import com.vaadin.flow.component.button.Button;
 import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
 
 public class MainMenu extends HorizontalLayout{
 
-    private final ToDoItemRepository myToDoRepo;
+    ToDoItemRepository myRepo;
 
-    MainMenu(ToDoItemRepository otherToDoRepo)
+    MainMenu(ToDoItemRepository otherToDoItemRepo)
```

Comment:
I wouldn't inject repositories directly into a view because of the transactional problem you had earlier, that's why you need DTO's and view models.